hack/update-generated-crd-code.sh

```diff
@@ -28,8 +28,9 @@ if [[ ! -d "${GOPATH}/src/k8s.io/code-generator" ]]; then
   exit 1
 fi
 
-if [[ ! -d "${GOPATH}/src/sigs.k8s.io/controller-tools" ]]; then
-  echo "sigs.k8s.io/controller-tools missing from GOPATH"
+if ! command -v controller-gen > /dev/null; then
+  echo "controller-gen is missing"
+  echo "please retry after running the following command locally: go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.4"
```

Comment:
Should we used the lated release v0.3.0?
https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.3.0

Reply:
We‘ve always been using v0.2.4. So, prefer to continue using it until we have to upgrade.